test/Dialect/XTenNN/subgraph-invalid.mlir

```diff
@@ -0,0 +1,58 @@
+// RUN: aten-opt %s -split-input-file -verify-diagnostics
+
+func.func @missing_terminator() -> f64 {
+    // expected-error@+1 {{enclave is missing terminator}}
```

Comment:
NIT: I find the use of `enclave` in the error message confusing as it doesn't refer to anything in the source. I suggest dropping the term from the message as I think `missing terminator` with the location is enough. (Similar for the other messages.) But I wouldn't want to fight about this, and cave easily to other opinions.

Reply:
I agree. Also less noise.

Reply:
Fixed this.